extensions/sql-database-projects/src/common/azureFunctionsUtils.ts

```diff
@@ -41,6 +41,36 @@ export async function getLocalSettingsJson(localSettingsPath: string): Promise<I
 	};
 }
 
+/**
+ * Adds a new setting to a project's local.settings.json file
+ * modified from setLocalAppSetting code from vscode-azurefunctions extension
+ * @param projectFolder full path to project folder
+ * @param key Key of the new setting
+ * @param value Value of the new setting
+ * @returns true if successful adding the new setting, false if unsuccessful
+ */
+export async function setLocalAppSetting(projectFolder: string, key: string, value: string): Promise<boolean> {
```

Comment:
Should project folder be a string, or URI?

Reply:
The AF extension version of this function takes a string, but I can change it to be a uri if that's better practice

Reply:
I have such mixed feelings on this.  I feel we need a version of the`path` package that can operate on `vscode.Uri` objects instead of just strings, 'cause I don't like having URIs in generic string format longer than they need to, but then you can't math on them.

Reply:
We should really try to use URI whenever possible and only convert to a string when it's absolutely necessary (such as passing to path functions).